README.md

```diff
@@ -1,2 +1,233 @@
+>>> pyminio_client.cp('/foo1/bar', '/foo2/')
+>>> pyminio_client.cp('/foo/bar/', '/foo/', recursive=True)
+>>> pyminio_client.cp('/foo1/bar/', '/foo2/', recursive=True)
+>>> pyminio_client.cp('/foo1/bar/', '/foo2/baz', recursive=True)
+ValueError: can not activate this method from directory to a file.
+```
+
+### mv
+Pyminio.mv works like [cp](#cp) only that it remove the source after complete the transfare. Works like linux's 'mv (-r)'.
+
+This func can only move recursively when the recursive flag is True. If not, it will raise a ValueError.
+
+```python
+>>> pyminio_client.mv('/foo/bar/', '/foo/baz/')
+```
+
+### get
+Pyminio.get return an object from given path. This object will return as a pyminio.File object or an pyminio.Folder object, that both inherit from pyminio.ObjectData
+
+This objects will contain metadata, their path and name.
+If its a File object it will contains it's data.
+
+```python
+>>> pyminio_client.get('/foo/bar/baz')
+File(name='baz', full_path='/foo/bar/baz', metadata=AttrDict({'is_dir': False, 'last_modified': time.struct_time(...), 'size': ..., 'content-type': ...}), data=...)
+```
+
+### get_last_object
+Pyminio.get_last_object will return the last modified object inside a given directory.
+
+This function must get a directory path or it will raise a ValueError.
+
+```python
+>>> pyminio_client.get_last_object('/foo/bar/')
+File(name='baz', full_path='/foo/bar/baz', metadata=AttrDict({'is_dir': False, 'last_modified': time.struct_time(...), 'size': ..., 'content-type': ...}), data=...)
+```
+
+### put_data
+Pyminio.put_data gets a path, data in bytes, and some metadata, and create an object inside the given path.
+
+```python
+>>> data = b'test'
+>>> metadata = {'Pyminio-is': 'Awesome'}
+>>> pyminio_client.put_data(path='/foo/bar/baz', data=data, metadata=metadata)
+```
+
+### put_file
+Pyminio.put_file works like [put_data](#put_data) only that instead of data it gets a path to a file in you computer. Then it will copy this file to the given location.
+
+```python
+>>> metadata = {'Pyminio-is': 'Awesome'}
+>>> pyminio_client.put_data(path='/foo/bar/baz', file_path='/mnt/some_file', metadata=metadata)
+```
+
+## Contribute
+
+All contributions are welcome:
+
+- Read the [issues](https://github.com/mmm1513/pyminio/issues), Fork the [project](https://github.com/mmm1513/pyminio) and do a Pull Request.
+- Request a new topic creating a `New issue` with the `enhancement` tag.
+- Find any kind of errors in the code and create a `New issue` with the details and the `bug enhancement` or fork the project and do a Pull Request.
+- Suggest a better or more pythonic way for existing examples.
+
+### Work environment:
+
+After forking the project and installing the dependencies,
+download the [minio docker](https://hub.docker.com/r/minio/minio/) and start an instance in your computer for development and testing.
+#### Dont forget to write tests, and to run all the tests before making a pull request.
```

Comment:
add PULL_REQUEST_TEMPLATE.md (in root repo) file. it will help alot with showing people what are the required steps
https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository